app/templates/components/donations/donation-progress.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="information">
+  <div class="amount">
+    <div class="value">{{format-currency amountDonated trimZero=true}}</div>
+    <div class="descriptive-text">given each month</div>
+  </div>
+  <div class="percentage">
+    <div class="value">{{format-percentage percentage trimZero=true}}</div>
+    <div class="descriptive-text">of goal</div>
+  </div>
+</div>
+{{progress-bar-container percentage=percentage}}
+<div class="description">{{donationGoal.description}}</div>
```

Comment:
Maybe it's the naming structure itself that I find problematic. Something feels wrong here every time I look at the conventions we're using in our styling.